Library/Homebrew/utils/github.rb

```diff
@@ -501,6 +500,26 @@ def check_for_duplicate_pull_requests(name, tap_remote_repo, state:, file:, args
     end
   end
 
+  def get_long_build_pull_requests_files(name, tap_remote_repo)
+    pull_requests = search_issues("org:#{name}", repo: tap_remote_repo, state: "open", label: "\"long build\"")
+    prs = pull_requests.map { |pull_request| pull_request["number"] }
+    files = []
+    prs.each do |pr|
+      get_pr_files(name, tap_remote_repo, pr).each do |pr_file|
+        files.append({ "file" => pr_file["filename"], "pr" => pr })
+      end
+    end
+    files
+  end
+
+  def get_pr_filenames(name, tap_remote_repo, pr)
```

Comment:
Same here, I'd suggest this logic is moved into `brew bump-formula-pr`.

Reply:
I moved this out of the GitHub file